tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -372,16 +416,38 @@ module.exports = {
      *
      * @param {string} fileName
      */
-    waitForFileVisible: function (fileName) {
-      const rowSelector = this.getFileRowSelectorByFileName(fileName)
+    waitForFileVisible: async function (fileName) {
       const linkSelector = this.getFileLinkSelectorByFileName(fileName)
-      return this
+
```

Comment:
should `filesListScrollToTop` be called here? its so often called with `waitForFileVisible`

Reply:
You're right. Will fix